README.md

```diff
@@ -29,16 +29,6 @@ This repository currently contains two main branches :
  * **master** : this branch contains a proof-of-concept implementation of the parser, that compiles to an executable JAR file.
  * **skript-library** : this branch contains no implementations, and its purpose is to be used as a base library for any skript-parser-related projects.
 
-The **master** branch includes the CLI of the project too.
-
-## Building
```

Comment:
You should probably link the CLI-project in this part and explain it a bit instead of removing it completely.

Reply:
I would like this to be changed as well.